synapse/config/appservice.py

```diff
@@ -132,6 +132,13 @@ def _load_appservice(hostname, as_info, config_filename):
         for p in protocols:
             if not isinstance(p, str):
                 raise KeyError("Bad value for 'protocols' item")
+
+    if as_info["url"] == "":
+        logger.info(
+            "(%s) Explicitly empty 'url' provided. This application service " +
+            "will not receive events or queries.",
```

Comment:
I'd probably start the line ``" will`` since it makes it easier to spot the spaces. Also I'd be tempted to omit the '+' operator.